app/controllers/chatops/slack_slash_command_list_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Chatops
+  class SlackSlashCommandListController < ::ApplicationChatopsController
+    def handle
```

Comment:
This should be treated as a minimal first pass to be iterated on. Slack does not turn channels into links when the channel name is used. Instead we need to use the channel name to look up the channel ID and use that, like `"<##{grouping.channel_id}>`. I'm opting to leave this out of the current PR since we're planning to move to a DB backed model where we can cache the channel id.